input/tar2/tar_input.go

```diff
@@ -0,0 +1,163 @@
+package tar2
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha512"
+	"encoding/base64"
+	"hash"
+	"io"
+	"os"
+	"path"
```

Comment:
No, it shouldn't.  It's being used to clean paths fed into hashing.  The canonical form of paths for all internal handling is '`/`' -- Hashes should not be different on platforms with unusual opinions about path formatting.